src/internal/connector/exchange/service_restore.go

```diff
@@ -119,7 +130,30 @@ func RestoreExchangeEvent(
 		return nil, errors.New("msgraph event post fail: REST response not received")
 	}
 
-	return EventInfo(event), nil
+	for _, attach := range attached {
+		uploader := &eventAttachmentUploader{
+			calendarID: destination,
+			userID:     user,
+			service:    service,
+			itemID:     *response.GetId(),
+		}
```

Comment:
nit: should be safe to create this outside the loop and reuse a single instance

Reply:
Resolved [a234ee4](https://github.com/alcionai/corso/pull/1345/commits/a234ee42cdfeaf1fa619a1546ffa946fd93b642a)